tests/contrib/executors/test_kubernetes_executor.py

```diff
@@ -161,6 +165,39 @@ def test_worker_configuration_no_subpaths(self):
                     "subPath shouldn't be defined"
                 )
 
+    @mock.patch.object(conf, 'get')
+    @mock.patch.object(configuration, 'as_dict')
+    def test_worker_configuration_auth_both_ssh_and_user(self, mock_config_as_dict, mock_conf_get):
+        def get_conf(*args, **kwargs):
+            if(args[0] == 'core'):
+                return '1'
+            if(args[0] == 'kubernetes'):
+                if(args[1] == 'airflow_configmap'):
+                    return 'airflow-configmap'
+                if(args[1] == 'git_ssh_key_secret_name'):
+                    return 'airflow-secrets'
+                if(args[1] == 'git_ssh_key_secret_key'):
+                    return 'gitSshKey'
+                if(args[1] == 'git_user'):
+                    return 'some-user'
+                if(args[1] == 'git_password'):
+                    return 'some-password'
+                if(args[1] == 'git_repo'):
+                    return 'git@github.com:apache/airflow.git'
+                if(args[1] == 'git_branch'):
+                    return 'master'
+                if(args[1] == 'git_dags_folder_mount_point'):
+                    return '/usr/local/airflow/dags'
+                if(args[1] == 'delete_worker_pods'):
+                    return True
+                return '1'
+            return None
+
+        mock_conf_get.side_effect = get_conf
+        mock_config_as_dict.return_value = {'core': ''}
+        with self.assertRaises(AirflowConfigException):
+            KubeConfig()
```

Comment:
What error are we expecting here? This test feels like it would be hard to debug if it breaks in the future. Either a check on the message, and/or a comment saying _why_ we expect this to raise please.

Reply:
is it better?

Reply:
Thanks.